cmd/minishift/cmd/delete.go

```diff
@@ -61,6 +60,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 	}
 
 	if clearCache {
+		fmt.Printf("Deleting Minishift ISO, OpenShift container images and OpenShift origin binaries from %s/cache\n", constants.Minipath)
```

Comment:
@dharmit @hferentschik I think we would be fine with the absolute path of the cache directory as mentioning individual files will make the output cluttered. I modified the code little bit (as below) and it looks fine to me.
```
if clearCache {
		cachePath := constants.MakeMiniPath("cache")
		err := os.RemoveAll(cachePath)
		if err != nil {
			atexit.ExitWithMessage(1, fmt.Sprintf("Error deleting Minishift cache: %v", err))
		} else {
			fmt.Printf("Removed the cache at: %s\n", cachePath)
		}
	}
```
I moved the print statement after deletion as I think saying deleting the cache would means we should again mention that it is deleted successfully.  Also as user has explicitly used --clear-cache we can just mention after deleting and no need to say we are deleting the cache. 
```
$ minishift delete --clear-cache
Removed the cache at: /home/lmohanty/.minishift/cache
Deleting the Minishift VM...
Minishift VM deleted.
```